code/modules/mob/living/basic/friendly/lizard.dm

```diff
@@ -25,41 +31,48 @@
 	butcher_results = list(/obj/item/food/meat = 1)
 	mob_biotypes = MOB_ORGANIC | MOB_BEAST | MOB_REPTILE
 	gold_core_spawnable = FRIENDLY_SPAWN
-	var/eating_sound = 'sound/weapons/bite.ogg'
+	/// List of stuff that we want to eat
+	var/static/list/edibles = list(
+		/mob/living/basic/butterfly,
+		/mob/living/basic/cockroach,
+		/obj/structure/spider/spiderling
+	)
 	/// Lizards start with a tail
 	var/has_tail = TRUE
 
-/mob/living/simple_animal/lizard/Initialize(mapload)
+/mob/living/basic/lizard/Initialize(mapload)
 	. = ..()
 	AddElement(/datum/element/wears_collar)
+	AddElement(/datum/element/basic_eating, heal_amt_ = 5, food_types_ = edibles)
+	ai_controller.set_blackboard_key(BB_BASIC_FOODS, typecacheof(edibles))
 
-/mob/living/simple_animal/lizard/handle_automated_action()
-	if(stat == CONSCIOUS && !buckled)
-		if(prob(1))
-			custom_emote(EMOTE_VISIBLE, pick("sticks out its tongue.", "wags its tail.", "lies down."))
+/datum/emote/lizard
+	mob_type_allowed_typecache = /mob/living/basic/lizard
+	mob_type_blacklist_typecache = list()
 
-	if(!isturf(loc))
-		return
-	var/list/lizard_view = view(1, src)
-	for(var/mob/living/M in lizard_view)
-		if(M.stat == CONSCIOUS && Adjacent(M) && HAS_TRAIT(M, TRAIT_EDIBLE_BUG))
-			custom_emote(EMOTE_VISIBLE, "eats \the [M]!")
-			playsound(loc, eating_sound, 20, 1)
-			M.death()
-			break
-	for(var/obj/structure/spider/spiderling/spider in lizard_view)
-		if(Adjacent(spider) && HAS_TRAIT(spider, TRAIT_EDIBLE_BUG))
-			if(prob(90)) // Slippery things aren't they?
-				visible_message("<span class='notice'>The spiderling skitters away from the [src]!</span>")
-				spider.random_skitter()
-				return
-			else
-				custom_emote(EMOTE_VISIBLE, "eats \the [spider]!")
-				playsound(loc, eating_sound, 20, 1)
-				qdel(spider)
-				break
+/datum/emote/lizard/whicker
+	key = "tongue"
+	message = "sticks its tongue out contentedly!"
+	emote_type = EMOTE_VISIBLE | EMOTE_AUDIBLE
```

Comment:
Can you move this emote code to where the current simple_animal emote code is, or maybe make a new file for basic mob emotes? I'd prefer if they're not spread out like this.